ros/src/computing/perception/detection/packages/lidar_tracker/nodes/obj_fusion/obj_fusion.cpp

```diff
@@ -123,151 +151,203 @@ static void fusion_objects(void) {
   pub_msg.header = header;
   autoware_msgs::CloudClusterArray cloud_clusters_msg;
   cloud_clusters_msg.header = header;
+  visualization_msgs::MarkerArray marker_array_msg;
+  int marker_id = 0;
 
-  for (unsigned int i = 0; i < obj_label_current.obj_id.size(); ++i) {
-    jsk_recognition_msgs::BoundingBox bounding_box;
-    if (obj_indices.at(i) == -1)
-      continue;
-
-    v_cloud_cluster_current.at(obj_indices.at(i)).label = object_type;
+  for (unsigned int i = 0; i < obj_label.obj_id.size(); ++i) {
+    if (obj_indices.at(i) == -1) continue;
 
+    v_cloud_cluster.at(obj_indices.at(i)).label = object_type;
     if (object_type == "car") {
-      v_cloud_cluster_current.at(obj_indices.at(i)).bounding_box.label = 0;
+      v_cloud_cluster.at(obj_indices.at(i)).bounding_box.label = 0;
     } else if (object_type == "person") {
-      v_cloud_cluster_current.at(obj_indices.at(i)).bounding_box.label = 1;
+      v_cloud_cluster.at(obj_indices.at(i)).bounding_box.label = 1;
     } else {
-      v_cloud_cluster_current.at(obj_indices.at(i)).bounding_box.label = 2;
-      v_cloud_cluster_current.at(obj_indices.at(i)).label = "unknown";
+      v_cloud_cluster.at(obj_indices.at(i)).bounding_box.label = 2;
+      v_cloud_cluster.at(obj_indices.at(i)).label = "unknown";
+    }
+
+    jsk_recognition_msgs::BoundingBox bounding_box;
+    bounding_box = v_cloud_cluster.at(obj_indices.at(i)).bounding_box;
+
+    /* adjust object rotation using lane in vector_map */
+    tf::Quaternion q_obj(bounding_box.pose.orientation.x,
+      bounding_box.pose.orientation.y,
+      bounding_box.pose.orientation.z,
+      bounding_box.pose.orientation.w);
+    bool fixed_rotation = false;
+    if (use_vmap) {
```

Comment:
If 'use_vmap' is false, the object is not rotated and the result of euclidean cluster is used as it is.